year/2016/16EE.ipynb

```diff
@@ -1051,4 +1050,146 @@
     },
     {
      "data": {
-      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAW4AAAD8CAYAAABXe05zAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDIuMi4yLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvhp/UCwAADONJREFUeJzt3X2sZPVdx/H3R5aWB7E05VZr4XpL0qC1sQVvCC0JQWgq7RqITwkkrZJINo3WUGNi1piY1H/ExBjjQzRrqeJTn7AY5KGCabE2EeouBbp0aaTttkWoQJtC0aSV+vWPmW1uxrl3znJn7twvvF/JZmfm/O6cz/723A/n/uacJVWFJKmP71p2AEnS8bG4JakZi1uSmrG4JakZi1uSmrG4JakZi1uSmrG4JakZi1uSmtmziDc944wzam1tbRFvLUnPS4cOHXqyqlaGjF1Ica+trXHw4MFFvLUkPS8l+eLQsS6VSFIzFrckNWNxS1IzFrckNWNxS1Izg64qSXIU+AbwbeDZqlpfZChJ0uaO53LAH6uqJxeWRJI0iEslktTM0OIu4I4kh5LsW2QgSdLWhi6VXFhVjyZ5OXBnkoeq6uMbB4wLfR/A6urqnGNKy7e2/9Yttx+9bu8OJdEL3aAz7qp6dPz748BNwPlTxhyoqvWqWl9ZGXS7vSTpOZhZ3ElOTXLascfAm4HDiw4mSZpuyFLJ9wI3JTk2/m+r6iMLTSVJ2tTM4q6qzwOv24EskqQBvBxQkpqxuCWpGYtbkpqxuCWpGYtbkpqxuCWpGYtbkpqxuCWpGYtbkpqxuCWpGYtbkpqxuCWpGYtbkpqxuCWpGYt
```

Comment:
this part is strange to me. i do not know what the values means .
 i wanted to update the updated part with plot but it seems not working in git hub repo.
check your mail for the jupyter file